bindings/java/src/main/native/context.cc

```diff
@@ -9,6 +9,46 @@ extern "C" {
 #include <ucp/core/ucp_mm.h>
 }
 
+/**
+ * Iterates through entries of java's hash map and apply
+ * ucp_config_modify and ucs_global_opts_set_value to each key value pair.
+ */
+static inline void iterate_java_map(JNIEnv *env, ucp_config_t *config,
```

Comment:
1. why does it have to be inline?
2. rename to jucx_apply_config_map() or similar